packages/mermaid-extension/src/index.ts

```diff
@@ -65,13 +66,14 @@ const contextCommands: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/mermaid-extension:context-commands',
   description: 'Provides context menu commands for mermaid diagrams.',
   autoStart: true,
-  requires: [IMermaidManager],
-  activate: (app: JupyterFrontEnd, mermaid: IMermaidManager) => {
+  requires: [IMermaidManager, ITranslator],
```

Comment:
Thank you for opening a PR quickly! Could you make `ITranslator` optional and fallback to `nullTranslator` to allow the plugin to be used with translator service disabled? For example see:

https://github.com/jupyterlab/jupyterlab/blob/bd46c76046671ae375b3ddd207ca441841e57123/packages/tooltip-extension/src/index.ts#L45-L51

Reply:
Done.